integreat_cms/cms/models/pages/page.py

```diff
@@ -256,7 +256,7 @@ def archived(self) -> bool:
         """
         return self.explicitly_archived or self.implicitly_archived
 
-    def get_non_archived_children(self) -> Iterator[Page]:
+    def get_non_archived_children(self, include_parent: bool | None = True) -> Iterator[Page]:
```

Comment:
```suggestion
    def get_non_archived_children(self, include_parent: bool = True) -> Iterator[Page]:
```
If an optional parameter has a default value that is not `None`, we don't need the `| None` as type hint: Either, `True`/`False` is directly passed, then it's a `bool`, or it is omitted, then it is set to `True` which is also a `bool`. So the variable `include_parent`  is always a `bool` and never `None`.